lightning/src/ln/channel.rs

```diff
@@ -210,6 +210,20 @@ const MULTI_STATE_FLAGS: u32 = (BOTH_SIDES_SHUTDOWN_MASK | ChannelState::PeerDis
 
 const INITIAL_COMMITMENT_NUMBER: u64 = (1 << 48) - 1;
 
+/// Liveness is called to fluctuate given peer disconnecton/monitor failures/closing.
+/// If channel is public, network should have a liveness view announced by us on a
+/// best-effort, which means we may filter out some status transitions to avoid spam.
+/// See further timer_chan_freshness_every_min.
+#[derive(PartialEq)]
+enum UpdateStatus {
```

Comment:
I find the constant names here confusing, can we use like "Live", "DisabledUnannounced", and "DisabledAnnounced" or so, mostly cause I dont want to look up the constant docs to figure out what to_dirty() and to_fresh() means.